openedx/features/personalized_learner_schedules/show_answer/show_answer_field_override.py

```diff
@@ -3,7 +3,9 @@
 new Show Answer values that remove the Past Due check (keeping the rest intact)
 """
 
-from common.lib.xmodule.xmodule.capa_module import SHOWANSWER
+from django.conf import settings  # lint-amnesty, pylint: disable=unused-import
+
```

Comment:
Why was this added?

Reply:
@robrap Because I rebased haphazardly. Thanks for catching! 😄